mlflow/store/sqlalchemy_store.py

```diff
@@ -465,7 +465,7 @@ def set_tag(self, run_id, tag):
 
     def delete_tag(self, run_id, key):
         """
-        Delete a tag from a run.
+        Delete a tag from a run. This is irreversible.
         :param run_id: ID of the run that contains the tag to delete
         :param key: name of the tag to delete
         """
```

Comment:
name of the tag to delete -> Name of the tag to delete

Reply:
Resolved in previous PR